### PR TITLE
Eausadha API logic modified

### PR DESCRIPTION
**Pull request by indraniBan**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.  

# Type of change  

- [ ] Bug fix  
- [ ] New feature  
- [ ] Enhancement  
- [ ] Refactoring  
- [ ] Documentation  
- [ ] Other ( please specify )  

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ravishanigarapu — APPROVED** [review]
looks fine, but we need to think about eaushada authorization where we need to keep.
As of now am approving this pr because its Its GET API .
@drtechie  please review and approve

**Review by ravishanigarapu — COMMENTED** [review]
Please resolve conflicts

File: src/main/environment/common_ci.properties [hunk 1]
```diff
@@ -98,3 +98,5 @@ eSanjeevani.routeUrl: @env.ESANJEEVANI_ROUTE_URL@
 biometric.discover.url = @env.BIOMETRIC_URL@
 biometric.deviceInfo.url = @env.BIOMETRIC_URL@/rd/info
 biometric.capture.url = @env.BIOMETRIC_URL@/rd/capture
+eAusadhaUrl=https://dlc.kar.nic.in/e-services/api/DWInstituteInward
+authorization=Basic a3Ntc2NsOktzbXNjbCMxMjM=
```
Comment (drtechie) [inline src/main/environment/common_ci.properties:102]: Can you please make this key more specific? "authorization" is not explanatory at application level.
"eausadha-authorization" maybe?

Comment (drtechie) [reply src/main/environment/common_ci.properties:102]: Also, can we please remove this key from committing to public?

Comment (indraniBan) [reply src/main/environment/common_ci.properties]: resolved the conflict and modified these properties
please review and merge it @drtechie  @ravishanigarapu

**Review by ravishanigarapu — APPROVED** [review]
looks fine

**Pull request merged**